app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 class='forget-pass'>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
```

Comment:
- [ ] I would kindly suggest we follow the project requirement as much as possible as this will show confidence in your ability to replicate any given design following the requirement strictly. Currently, the design of your `registration and login` page does not look like what we have in the original wireframe design and I would suggest you check the wireframe and replicate the same design to yours.

| Your design 👇| Wireframe |
| ------------- | ------------- |
| <img width="960" alt="log" src="https://github.com/techmoves/movesBloc/assets/78634841/4b9833c1-8022-4c0b-9cad-f943e35b41f9"> | ![blog_login](https://github.com/techmoves/movesBloc/assets/78634841/bf348564-95cd-469e-85c6-751bc8fcb48d)|